src/readfilter.hpp

```diff
@@ -1622,7 +1628,7 @@ inline void ReadFilter<Alignment>::emit_tsv(Alignment& read, std::ostream& out)
                 }
             }
         } else {
-            cerr << "I didn't implement all fields for tsv's so if I missed something let me know and I'll add it -Xian" << endl;
+            cerr << endl << "Available fields: <https://github.com/vgteam/vg/wiki/Getting-alignment-statistics-with-%E2%80%90%E2%80%90tsv%E2%80%90out>" << endl;
```

Comment:
We might be able to get away with the non-percent-encoded versions of these dash characters here. But then we have special dashes that look identical to normal dashes but aren't.

Reply:
That might make more sense; right now the page is named after the option but the option only applies to one particular subcommand, which I thought was a little weird.